v3.1

backsum opened the pull request:
1. Added **validation** of input format being time;
2. Added **exit message** if user entered something other than time;
3. Made **comments** about the code's logic **clearer**.
4. Changed prompt for input with **clarification of time format**.

Commit:
v2

Commit:
v3

1. Added validation of input format being time;
2. Added exit message if user entered something other than time.
3. Made comments about the code's logic clearer.

Anna-ghar commented:
The way you've parsed the input time and computed the remaining seconds is efficient. Your code also validates the input format, ensuring the countdown works as expected.Well done!

Commit:
v3.1

1. Added validation of input format being time;
2. Added exit message if user entered something other than time;
3. Made comments about the code's logic clearer;
4. Changed prompt for input with clarification of time format.
